src/inventory_ui.cpp

```diff
@@ -295,13 +337,38 @@ void inventory_selector::print_middle_column() const
     print_column(worn, middle_column_offset, middle_column_width, selected_w, current_page_offset_w);
 }
 
-void inventory_selector::print_left_column() const
+void inventory_selector::print_left_column(const bool show_direction) const
 {
-    print_column(items, left_column_offset, left_column_width, selected_i, current_page_offset_i);
+    print_column(items, left_column_offset, left_column_width, selected_i, current_page_offset_i, show_direction);
+}
+
+std::string suffix_direction ( int direction ) {
```

Comment:
Interesting, it's an int, but it's compared to values of the `aim_direction` enum. But why `aim_direction`, what has it to do with the advanced inventory manager?

There is another enum: `direction`, which may be more appropriate. Both enums contain values that you don't need (`aim_direction` has AIM_CONTAINER and AIM_ALL, `direction` has the whole above and below stuff), but you can use `direction_name_short` to get the abbreviation for a `direction` instead of re-implementing it here.